docs/core-concepts/universal-commerce-schema/index.md

```diff
@@ -1239,3 +1254,56 @@ erDiagram
     }
 ```
 
+## Key Components
+
+### Orders and Order Line Items
+
+- **Orders**: Represents individual customer orders, capturing details such as order ID, customer ID, market ID, store ID, payment ID, order total, order status, and timestamps.
+- **Order Line Items**: Represents items within an order, capturing details such as order item ID, product variant store ID, price, tax, shipping, discount, and quantity.
+
+### Product Listings and Products
+
+- **Product Listings**: Represents listings of products available for sale in different markets and stores, capturing details such as product list ID, market ID, store ID, price, and status.
+- **Products**: Represents individual products, capturing details such as product ID, name, type, tags, supplier ID, category, brand, and status.
+- **Product Variants**: Represents variations of a product, capturing details such as variant ID, SKU, color, size, weight, and status.
+
+### Stores and Inventory History
+
+- **Stores**: Represents physical or online store locations, capturing details such as store ID, market ID, location, address, and contact information.
+- **Inventory History**: Represents historical changes in inventory levels for product variants in stores, capturing details such as product variant store ID, inventory quantity, and timestamps.
```

Comment:
It's a periodic (daily) snapshot of inventory level as opposed to a change in inventory level

Reply:
Updated, re-requesting review